matminer/data_retrieval/retrieve_MDF.py

```diff
@@ -35,76 +34,67 @@ def __init__(self, anonymous=False, **kwargs):
 
         self.forge = Forge(anonymous=anonymous, **kwargs)
 
-    def get_dataframe(self, source_names=None, elements=None, titles=None,
-                      tags=None, resource_types=None, match_fields=None,
-                      exclude_fields=None, match_ranges=None,
-                      exclude_ranges=None, unwind_arrays=True):
+    def api_link(self):
+        return "https://materialsdatafacility.org/"
```

Comment:
Could you change this to: https://github.com/materials-data-facility/forge?

We're working on a readthedocs, and I'll point to that once it's complete.

Reply:
Ok its been changed